react-paw-mailmerge/src/components/table/TableOutput.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import Table from "react-bootstrap/Table";
+import TableHead from "./TableHead";
+import TableRow from "./TableRow";
+import Pagination from 'react-bootstrap/Pagination';
+
+interface TableOutputProps {
+    data: Record<string, any>[];
+    template: string;
+}
+
+const TableOutput: React.FC<TableOutputProps> = ({data}) => {
+    // pagination
+    const [currentPage, setCurrentPage] = useState<number>(1);
+    const handlePageChange = (page: number) => {
+        setCurrentPage(page);
+    };
+
+    const ItemsPerPage = 10;
```

Comment:
What are your thoughts on having the numeric constants towards the top of the page?

Reply:
Good catch. It makes sense to move it toward the top.